.chloggen/cicd-reg-attr.yaml

```diff
@@ -0,0 +1,24 @@
+# Use this changelog template to create an entry for release notes.
+#
+# If your change doesn't affect end users you should instead start
+# your pull request title with [chore] or use the "Skip Changelog" label.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: breaking
+
+# The name of the area of concern in the attributes-registry, (e.g. http, cloud, db)
+component: cicd, deployment
```

Comment:
this and corresponding names in github templates should be updated as soon as there will be clear structure if new namespaces go under cicd parent of it they should be standalone namespaces.

Reply:
Opened #1194